projects/plugins/jetpack/modules/theme-tools/content-options/blog-display.php

```diff
@@ -33,6 +33,10 @@
  */
 function jetpack_blog_display_custom_excerpt( $content ) {
 	$post = get_post();
+	if ( empty( $post ) ) {
+		return $content;
```

Comment:
I wonder if this may cause issues in some cases since we check for `empty()` later:
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/theme-tools/content-options/blog-display.php#L120-L125

Maybe we should extend that check to avoid displaying the full content twice here?
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/theme-tools/content-options/blog-display.php#L127

Reply:
Good catch, I missed that somehow.

I guess `jetpack_blog_display_custom_excerpt()` probably just shouldn't accept a `$content` param and should just return an empty string instead.

I'll update the PR.